testbench/common/prbs_gen_for_sys_id_tb/Manifest.py

```diff
@@ -0,0 +1,10 @@
+files = [
+    "prbs_gen_for_sys_id_tb.vhd",
+]
+
+modules = {
+    "local" : [
+        "../../../modules/common/prbs_gen/",
```

Comment:
Please use the root Manifest.py here only. Make sure that both `prbs_gen` and `prbs_for_sys_id` are in the `modules/common/Manifest.py` file.